### PR TITLE
add: change language Signup

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add change language in signup. replicate for login

### CODE REVIEW

File: src/components/Signup/Signup.js [hunk 1]
```diff
@@ -81,15 +81,36 @@ export default injectIntl(function({ intl }) {
...
+                </Link>
+              </div>
+            )}
+          </div>
         </header>
         <h5>{_('signup.sign_up')}</h5>
         <p className="content-subtitle">{_('signup.sign_up_sub')}</p>
+        <p className="content-subtitle">
+          {_('signup.do_you_already_have_an_account')}{' '}
+          <Link to="/login" className="link-green uppercase">
```
Comment (andresmoschini) [inline src/components/Signup/Signup.js]:
```suggestion
          <Link to="/login" className="uppercase">
```
?

Comment (fcoronelmakingsense) [reply src/components/Signup/Signup.js]: image


it's this green

Comment (andresmoschini) [reply src/components/Signup/Signup.js]: Yes, almost all our links are green.

Comment (fcoronelmakingsense) [reply src/components/Signup/Signup.js]: Yes i view it, i copy from html from doppler ui system. I delete

Comment (fcoronelmakingsense) [reply src/components/Signup/Signup.js]: Ready

**Comment by andresmoschini** [comment]
👍

File: src/components/Signup/Signup.js [hunk 2]
```diff
@@ -81,15 +81,36 @@ export default injectIntl(function({ intl }) {
...
         <header>
           <h1 className="logo-doppler-new">Doppler</h1>
-          <small className="content-signin">
-            {_('signup.do_you_already_have_an_account')}{' '}
-            <Link to="/login" className="link-green uppercase">
-              {_('signup.log_in')}
-            </Link>
-          </small>
+          <div className="language-selector">
+            {intl.locale === 'es' ? (
```
Comment (andresmoschini) [inline src/components/Signup/Signup.js]: Did you test if it is work fine after a change?

Comment (andresmoschini) [reply src/components/Signup/Signup.js]: I said that because `intl.locale` is not part of props or state, right?

Comment (fcoronelmakingsense) [reply src/components/Signup/Signup.js]: yes it's works. in the function has the dependency

**Pull request merged**